src/assets/scss/_array-field.scss

```diff
@@ -0,0 +1,135 @@
+.squished-array-buttons {
+  .array-input-field-container {
+    .add-button {
+      padding-right: 8px;
+    }
+    .input-items-new {
+      .item {
+        .action-buttons {
+          margin-right: -1px;
+        }
+      }
+    }
+  }
+}
+
+// New Styles
+.entity-value,
+.select-all-input,
+.multi-form-input {
+  .add-button {
+    padding-right: 6px;
+  }
+  .array-input-field-container {
+    ul {
+      padding-left: 0;
+    }
+    .add-element-container {
+      display: flex;
+      flex-direction: row;
+      gap: 5px;
+      [class*='input-switch-container'] {
```

Comment:
What is this rule trying to do?

Reply:
It ensures that the input field takes all the available width. This is how it looks with and without this rule.
![image](https://github.com/informatics-isi-edu/chaise/assets/44138408/f8a4697b-2723-45be-be58-415507a376e3)
![image](https://github.com/informatics-isi-edu/chaise/assets/44138408/919c4fb5-e288-4a8e-b8be-df15d663302f)